src/modules/shared/services/innovations.dtos.ts

```diff
@@ -179,3 +180,24 @@ export type InnovationActivityLogListDTO = {
       link: null | { label: string, url: string }
     })[]
 };
+
+export type InnovationStatisticsDTO = {
+  [InnovationStatisticsEnum.ACTIONS_TO_SUBMIT_COUNTER]: { count: number; total: number; lastSubmittedSection: null | string; lastSubmittedAt: null | DateISOType;},
+  [InnovationStatisticsEnum.SECTIONS_SUBMITTED_COUNTER]: { count: number; total: number; lastSubmittedSection: null | string; lastSubmittedAt: null | DateISOType;},
+  [InnovationStatisticsEnum.UNREAD_MESSAGES_COUNTER]: { count: number; lastSubmittedAt: null | DateISOType;},
+  [InnovationStatisticsEnum.ACTIONS_TO_REVIEW_COUNTER]: { count: number; lastSubmittedAt: null | DateISOType;},
+  [InnovationStatisticsEnum.SECTIONS_SUBMITTED_SINCE_SUPPORT_START_COUNTER]: { count: number; lastSubmittedAt: null | DateISOType;},
+}
+
+export type baseStatisticsCard = {
```

Comment:
@Emanuel-Mariano-BJSS I think this types should start with uppercase if you agree

Reply:
Agreed